spec/services/cart_service_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe CartService do
+  describe 'get user' do
+    context 'there is a cart with valid user' do
+
+      let!(:user) { create(:user) }
+      let!(:cart) { create(:cart, user: user) }
+
+      it 'should returns the user' do
+        cart_user = CartService.call(cart, nil)
+        expect(user).to eq(cart_user)
+      end
+    end
+
+    context 'there is a cart without valid user' do
+      it 'should returns the ' do
```

Comment:
the...? :sweat_smile:  

também não é recomendado utilizar `should` nas descrições de specs, pois utilizar `should` na descrição e em seguida `expect` no teste soa redundante.

Reply:
oops acabei esquecendo de completar, ah não tinha me ligado na questão do should, vou consertar